src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
```

Comment:
don't write this with `*` selector, `box-sizing` is the only good option to use with this selector